packable/packable/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "packable"
-version = "0.8.1"
+version = "0.8.2"
```

Comment:
Sounds significant enough to be a minor 🤷🏻

Reply:
Really? I don't think so. It's just more non-breaking impls and types.

Reply:
Minor doesn't necessarily mean breaking, but ok fine